examples/datasource-streaming-websocket/websocket-server/src/server.js

```diff
@@ -1,8 +1,10 @@
 const WebSocket = require("ws");
 
 const wss = new WebSocket.Server({ port: 8080 });
+console.log("WebSocket server listening on ws://localhost:8080/");
 
 wss.on("connection", (ws) => {
+  console.log("WebSocket connection established.");
```

Comment:
Were these console logs supposed to be pushed?

Reply:
This websocket server is just helper to test this plugin example and when we run it it is more user friendly to see that server is running. It logs it into the console